yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -141,8 +141,7 @@ func InitAssessmentDB() error {
 			mean_exec_time REAL,
 			min_exec_time REAL,
 			max_exec_time REAL,
-			rows		BIGINT,
-			PRIMARY KEY(queryid));`, DB_QUERIES_SUMMARY),
+			rows		BIGINT);`, DB_QUERIES_SUMMARY),
```

Comment:
Looks like there can be duplicates then. We should somehow detect and skip duplicates, otherwise they will come up in assessment report?

Reply:
Yes while queries the sqlite assessmentDB we have `DISTINCT` clause in the query
```
query := fmt.Sprintf("SELECT DISTINCT query from %s", migassessment.DB_QUERIES_SUMMARY)
```